torch/jit/__init__.py

```diff
@@ -1526,21 +1541,28 @@ def __init__(self, original, stubs):
             # __init__ can run correctly
             self.__dict__['_initialized'] = False
             super(WeakScriptModuleProxy, self).__init__()
-
             # Store a weak reference to the original module
             self.__dict__["_original"] = weakref.ref(original)
 
             constants_set = set(getattr(original, "__constants__", []))
             self.__dict__["_constants_set"] = {}
 
+            if not hasattr(original, '_parameters'):
```

Comment:
We need to document better exactly what happens when a we create a ScriptModule from a Module: what gets copied, what is translated in some way, what is ignored.